composer.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "paysafegroup/paysafe_sdk_php",
-	"version": "1.0.1",
```

Comment:
I didn't not include a specific version in `composer.json`, as I didn't want to potentially conflict with the release/tag name. Related: https://github.com/paysafegroup/paysafe_sdk_php/commit/2ba031de498ae5354ee1919ded54e25350283bca#commitcomment-22631188